app/settings.py

```diff
@@ -10,17 +11,28 @@
 logger = logging.getLogger(__name__)
 
 
+
 class Settings:
     def __init__(self):
         # Chat API - By default, we are assuming Oobabooga's Text Generation
         # WebUI is running
+        
+        # Guidance new settings
+        self.test_file = os.getenv("TEST_FILE", "/data/uploads/the_trial.txt")
+        self.embeddings_map = {
+            **{name: HuggingFaceInstructEmbeddings for name in ["hkunlp/instructor-xl", "hkunlp/instructor-large"]},
+            **{name: HuggingFaceEmbeddings for name in ["all-MiniLM-L6-v2", "sentence-t5-xxl", "/models/all-MiniLM-L6-v2"]}
+        }
+        self.persist_directory = os.getenv("PERSIST_DIRECTORY", "./persist_directory")
+
         self.embeddings_model = os.getenv(
-            "EMBEDDINGS_MODEL", "all-MiniLM-L6-v2"
+            "EMBEDDINGS_MODEL", "/models/all-MiniLM-L6-v2"
         )
 
         self.chat_api_url = os.getenv(
             "CHAT_API_URL", "http://0.0.0.0:5000/api/v1/generate"
         )
+        self.model_path = os.getenv("MODEL_PATH", "/models/wizardLM-7B-HF")
```

Comment:
If the model doesn't exist we need to provide a simple way for users to bootstrap